### PR TITLE
[DOCS] Add project board links to read me

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
### Description

Adds a link to the project read me pointing readers to the project tab for current release issues.

### Checklist
- [x] Code compiles correctly
- [x] All tests are passing
- [x] New tests have been created which fail without the change (if possible)
- [x] README / documentation was extended, if necessary
- [x] Changelog entry (if change is visible to the user)
- [x] Used meaningful commit messages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Please follow the
 [Developers Guide](https://solidity.readthedocs.io/en/latest/contributing.html)
 if you want to help.
 
+You can find our current feature and bug priorities for forthcoming release [in the projects section](https://github.com/ethereum/solidity/projects).
```
Comment (Marenz) [inline README.md]: `forthcoming releases` or `the forthcoming release`?

**Review by Marenz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ChrisChinchilla) [reply README.md]: Whoops, yes

```suggestion
You can find our current feature and bug priorities for forthcoming releases [in the projects section](https://github.com/ethereum/solidity/projects).
```

**Review by Marenz — DISMISSED** [review]

**Review by chriseth — CHANGES_REQUESTED** [review]
Please squash.

**Comment by Marenz** [comment]
We can use "Squash and merge"

**Comment by axic** [comment]
> We can use "Squash and merge"

Please don't. It doesn't leave a merge commit just pushes a commit on head :wink:

**Comment by Marenz** [comment]
sounds beautiful to me, I hate merge commits ;)

**Review by chriseth — APPROVED** [review]

**Pull request merged**